bin/kafka-run-class.sh

```diff
@@ -140,6 +140,11 @@ do
 done
 shopt -u nullglob
 
+if [ -z "$CLASSPATH" ] ; then
+  echo 'Classpath is empty. Please build the project first e.g. by running ''./gradlew jar -Pscala_version=$SCALA_VERSION'''
```

Comment:
I just tested this and it doesn't actually print the $SCALA_VERSION variable as I expected. I thought that the following would ensure it's set:

```sh
if [ -z "$SCALA_VERSION" ]; then
  SCALA_VERSION=2.11.11
fi
```

Can you please check?